client/windows/CAdvmapInterface.cpp

```diff
@@ -573,7 +573,7 @@ CAdvMapInt::CAdvMapInt():
 	strongInterest = true; // handle all mouse move events to prevent dead mouse move space in fullscreen mode
 	townList.onSelect = std::bind(&CAdvMapInt::selectionChanged,this);
 	bg = BitmapHandler::loadBitmap(ADVOPT.mainGraphic);
-	if (ADVOPT.worldViewGraphic != "")
+	if (!ADVOPT.worldViewGraphic.empty())

```

Comment:
We don’t use spaces after for if while pls fix here and in other placs too

Reply:
@Nordsoft91 fix it here https://github.com/vcmi/vcmi/pull/1111/commits/7fdad4e0f6edc1fae14b1008360f9bc2a9d885ef